exercicios/para-casa/projeto_I/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "projeto_i",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.18.2"
```

hmmm, acredito q algumas dependencias estao faltando aqui no package.json pra gente conseguir rodar o projeto certinho. Por exemplo: temos o express nas dependencias, mas nao temos o mongoose, bcrypt, dotenv-safe, jsonwebtoken etc. Coloquei aqui uma sugestão que pode funcionar pra gente:
```suggestion
    "express": "^4.18.2",
    "bcrypt": "^5.0.1",
    "cors": "^2.8.5",
    "dotenv-safe": "^8.2.0",
    "jsonwebtoken": "^8.5.1",
    "mongoose": "^6.0.12"
```